triveni-codev/GBWebApp-dev
# Darlp networks service extension 1203798041197117

Hi @triveni-codev .

Done implementing logics for genre, base url and language extension types for atsc3 service. These extension types basically have same implementation with those first 3 set of atsc3 service extension types here: https://github.com/triveni-codev/GBWebApp-dev/pull/529

ASANA: https://app.asana.com/0/1201943683850147/1203798041197117

PR Considerations:
-this is limited only for atsc 3 network type
-this is limited only for genre, base url and language extension types

To Test:
-adding and updating of atsc 3 service extensions on dirty change
-can retrieve any of these 3 existing service extension types from server, display and can be updated
-logics for the specified first 3 service extensions
>should allow any character inputs
>should only have one instance (i.e if genre is selected, it shouldn't be available in the dropdown options) even on existing record from server 
-big commit for adding and updating is already good

Special Note: For Language extension type, it will only work on big commit if you update the java codes on below files I specified.
